go.mod

```diff
@@ -10,4 +11,6 @@ require (
 	github.com/stretchr/testify v1.3.0
 )
 
+replace github.com/edgexfoundry/go-mod-registry => ../go-mod-registry
```

Comment:
Did you mean to commit this? Looks like a local development dependency.

Reply:
PR is in draft. go.mod will be updated before moving it out of draft.

Reply:
It's currently breaking the build for your PR.

```[2020-01-22T18:17:44.345Z] go: parsing /go-mod-registry/go.mod: open /go-mod-registry/go.mod: no such file or directory```

Reply:
Yep, That is expected.